spec/factories.rb

```diff
@@ -109,7 +109,7 @@
   factory :application_choice do
     association :application_form, factory: :completed_application_form
     course_option
-    status { ApplicationStateChange.valid_states.sample }
+    status { :application_complete }
```

Comment:
The factories should be as random as possible. If a test relies on the state being `application_complete`, it should explicitly set this up when creating the test object.